app/domain/authentication/authn_oidc/v2/commands/list_providers.rb

```diff
@@ -10,7 +10,9 @@ module Commands
         ListProviders ||= CommandClass.new(
           dependencies: {
             json_lib: JSON,
-            logger: Rails.logger
+            logger: Rails.logger,
+            repo: DB::Repository::AuthenticatorRepository,
```

Comment:
The `AuthenticatorRepository` shouldn't be injected here because it's a dependency of the `AuthenticatorRepository`.  Instead, pass the initialized `AuthenticatorRepository`:

```suggestion
            authenticator_repository: nil,
```
now in `ui.rake`
```ruby
Authentication::AuthnOidc::V2::Commands::ListProviders.new(
        logger: logger,
        authenticator_repository: DB::Repository::AuthenticatorRepository.new(
          logger: logger,
          data_object: Authentication::AuthnOidc::V2::DataObjects::Authenticator
        )
      ).call(
        message: passed_arguments
      )
```

**Note**: in general, please strongly prefer descriptive, non-abbreviated variable and argument names. It makes the code easier to understand.